client/src/commons/PhoneNumberTextField/PhoneNumberTextField.tsx

```diff
@@ -7,17 +7,16 @@ import CircleTextField, { Props as circleTextFieldProps } from 'commons/CircleTe
 const NOT_PHONE_ERROR = 'שגיאה: מספר שהוזן אינו תקין';
 
 const PhoneNumberTextField: React.FC<Props> = (props: Props): JSX.Element => {
-    const [isError, setIsError] = React.useState<boolean>(false);
 
-    const {id, value, onChange, testId, ...rest } = props;
+    const {id, value, onChange, testId, isValid, setIsValid, ...rest } = props;
 
     return (
         <CircleTextField
             {...rest}
-            error={isError}
-            onBlur={() => setIsError(value && !Validator.phoneValidation(value as string))}
-            onFocus={() => setIsError(false)}
-            label={isError && NOT_PHONE_ERROR}
+            error={!isValid}
+            onBlur={() => setIsValid(!value || Validator.phoneValidation(value as string))}
```

Comment:
phoneValidation shoud return false if (!value) is true anyway so !value condition is redundant

Reply:
this condition is to not show error when field is empty. !value will return true when value is empty, but phoneValidation will return false.

Reply:
but is an empty value valid?

Reply:
yes

Reply:
mandatory fields are only for after mvp, for now they are valid